### DISCUSSION TITLE
Zoom Meeting popups get hidden and very hard to find

### DISCUSSION THREAD

**Discussion by joefiorini** [qtile/qtile#3644] [Q&A]
My company uses Zoom heavily for meetings. When a meeting is recorded Zoom pops up a confirmation dialog. When that happens it blocks the rest of the user interface until you confirm, meaning I can't toggle mute or video while that dialog is present. The problem I'm having is that the confirmation dialog shows up UNDER the Zoom window and I can't switch to it at all, in fact I don't ever see it so I don't even know it's present. The only way to find the dialog is to drag the main window out of the way (and any other windows that might be covering it). I can't easily reproduce this since I need to be a guest on a recorded meeting, meaning I can't easily run `xsel` on it to get its properties.

Has anyone else had this problem? Is there something I can do in the config to keep this dialog on top? Also, does anyone know a way I can reproduce/test this?

UPDATE: This seems related to #1893, which was supposedly fixed by #1943. Seems like maybe it wasn't completely fixed?

**Comment by m-col** [comment]
I'm not sure how you can test it I'm afraid, but you could use a hook to ensure that any new floating windows are raised to the top of the stack, something like this (untested):
```py
from libqtile import hook

@hook.subscribe.client_managed
def _(window):
    if window.floating:
        window.cmd_bring_to_front()
```
You can also use `libqtile.log_utils.logger` to log information about it with `logger.warning(window.cmd_info())` which may provide useful information so you could target just that window to raise it.

It is pretty strange behaviour, so maybe that PR didn't fix it completely.
